app/views/sentences/several_random_sentences.ctp

```diff
@@ -102,9 +102,28 @@ $this->set('title_for_layout', $pages->formatTitle(__('Random sentences', true))
                 $sentence['Translation'],
                 $sentence['User']
             );
-        }
-        ?>
+    }
+    ?>
+
     </div>
 </div>
 
+<?php 
+} else {
+?>
+    <div id="main_content">
+        <div class="module">
+            <?php
+            if($searchProblem == 'disabled') {
+                echo $html->tag('h2', __('Search is currently disabled, please try again later.', true));
+            } else if ($searchProblem == 'error') {
+                echo $html->tag('h2', format(__('An error occurred while fetching random sentences. 
+                        If this persists, please <a href="{}">let us know</a>.', true),
```

Comment:
This string spans on two lines, which unnecessarily adds a newline and a lot of spaces between the two sentences. Please use concatenation instead:
```php
'First sentence. '. # note the leading space inside the string here
'And this is the '. # and here too
'second sentence'
```